packages/js/src/gen/livekit_webhook_pb.d.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright 2023 LiveKit, Inc.
```

Comment:
Nit: I think it's worth adding a `.gitattributes` file with something like this:

```
packages/js/src/gen/* linguist-generated=true
```

This would hint to GH to not show diffs for them